ui/app/src/Utils/DateTimePicker/Views.elm

```diff
@@ -80,14 +81,19 @@ viewMonth : DateTimePicker -> Posix -> Html Msg
 viewMonth dateTimePicker justViewTime =
     let
         days =
-            listDaysOfMonth justViewTime
+            listDaysOfMonth justViewTime dateTimePicker.firstDayOfWeek
 
         weeks =
             splitWeek days []
     in
     div [ class "row justify-content-center" ]
         [ div [ class "weekheader" ]
-            (List.map viewWeekHeader [ "Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat" ])
+            (if dateTimePicker.firstDayOfWeek == Sunday then
```

Comment:
Can we use `case of` here and other places where we check the firstDayOfWeek value. This way it would be clear that the only two options are Sunday and Monday.